locales/id.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Page": "Halaman",
+  "of": "dari",
+  "Published with": "Diterbitkan dengan",
+  "Subscribe": "Berlangganan",
+  "Subscribe to": "Berlangganan",
+  "Subscribed!": "Berlangganan!",
+  "Subscribe to our newsletter": "Berlangganan newsletter kami",
+  "You've successfully subscribed to": "Kamu telah berhasil berlangganan",
+  "Your email address": "Alamat email kamu",
+  "with the email address": "dengan alamat email",
+  "Email cannot be blank.": "Email tidak boleh kosong.",
+  "Get the latest posts delivered right to your inbox.": "Dapatkan postingan terbaru yang dikirim langsung ke kotak masuk kamu.",
+  "Stay up to date! Get all the latest & greatest posts delivered straight to your inbox.": "Tetap update! Dapatkan semua postingan terbaru & terkini yang dikirimkan langsung ke kotak masuk kamu.",
+  "Recommended for you": "Direkomendasikan untuk kamu",
+  "Featured": "Unggulan",
+  "Page not found": "Halaman tidak ditemukan",
+  "Unfortunately the page you were looking for could not be found.": "Sayangnya halaman yang kamu cari tidak dapat ditemukan.",
+  "Back to home": "Kembali ke beranda",
+  "Go to the home page": "Buka halaman beranda",
+  "Recent articles": "Artikel Terbaru",
+  "No recent articles found :(": "Tidak ada artikel terbaru yang ditemukan :(",
+  "1 min read": "1 menit baca",
+  "% min read": "% menit baca",
+  "No posts": "Tidak ada postingan",
+  "No posts found": "Tidak ada postingan yang ditemukan",
+  "Apparently there are no posts at the moment, check again later.": "Tampaknya tidak ada postingan saat ini, periksa lagi nanti.",
+  "1 post": "Satu postingan",
+  "% posts": "% postingan",
+  "Tags": "Tag",
+  "with this tag": "dengan tag ini",
+  "No tags found :(": "Tidak ada tag yang ditemukan :(",
+  "Posted by": "Dikirim oleh",
+  "Among with": "<br>Di antara dengan ",
+  "Among with no break line": "Di antara dengan ",
+  "Type to search": "Ketik untuk mencari",
+  "No results for your search, try something different.": "Tidak ada hasil untuk pencarian kamu, coba sesuatu yang berbeda.",
+  "JavaScript license information": "Informasi lisensi JavaScript",
+  "Toggle dark mode": "Beralih ke mode gelap"
+}
```

Comment:
Thanks for the PR! Only one thing, can you please add a newline at the end of the file?

Reply:
The line has been added :smiley:

Reply:
Thank you! :)